### PR TITLE
template: perform ddl in apply_config

### PR DESCRIPTION
**Pull request by printercu**
because `if opts.is_master` branch is never invoked in role.init during rolling update

### CODE REVIEW

**Comment by dokshina** [comment]
Please, fix luacheck errors.

**Comment by printercu** [comment]
Fixed. Please, squash commits when merging.

File: examples/getting-started-app/app/roles/storage.lua [hunk 1]
```diff
@@ -135,27 +135,27 @@ local exported_functions = {
...
             box.schema.func.create(name, {if_not_exists = true})
             box.schema.role.grant('public', 'execute', 'function', name, {if_not_exists = true})
         end
     end
-
-    for name, func in pairs(exported_functions) do
-        rawset(_G, name, func)
-    end
-
-    return true
```
Comment (dokshina) [inline examples/getting-started-app/app/roles/storage.lua:152]: In the Cartridge application template both `init` and `apply_config` functions return `true`. 
This role is an example based on template, I think it should use the same convention.

The same is about function args, I suggest using `local function init(opts) -- luacheck: no unused args`, as it's defined in the template.

Comment (printercu) [reply examples/getting-started-app/app/roles/storage.lua:152]: Return values of this functions are not used

Comment (dokshina) [reply examples/getting-started-app/app/roles/storage.lua:152]: OK, I will remove return values from template, but renaming `opts` -> `options` confuses user. Example application is an edited template application, I don't see any reason to rename this parameter.

**Comment by dokshina** [comment]
See #428

**Pull request closed without merging**